bluesky/plan_stubs.py

```diff
@@ -908,6 +908,32 @@ def move():
     return (yield from trigger_and_read(list(detectors) + [motor]))
 
 
+def move_per_step(detectors, step, pos_cache):
+    """
+    Just the motion part of an N-dimensional step scan.
+
+    This can be used as a building block for custom ``per_step`` stubs.
+
+    Parameters
+    ----------
+    detectors : iterable
```

Comment:
you're not using `detectors` here.
I'm guessing you have a case where the overriding function does something with the detectors?
if yes, is it something that can't be done while staging or triggering?
I'd like to see an example

Reply:
Good catch. This is here because `(detectors, step, pos_cache)` is the required signature for a `per_step` plan. However, this stub shouldn't be used on it's own, so it isn't beholden to the same requirements as a normal `per_step` plan.

If you agree that nobody will be passing this into `per_step` on its own I'm happy to remove the arg. Passing this in by itself would result in a scan that moves some motors around and doesn't take any data.

Reply:
ah ok, no prob! I think we should wait to hear from @danielballan and @tacaswell what they think. There could be another reason why they kept `move()` enclosed.
I think this would be useful and the naming seems sensible.

Reply:
I vote for removing the detectors argument. It's easy enough to wrap this in a thing that accepts and ignores a detectors argument, and better to force the user to do that for clarity's sake.